presto-cache/src/main/java/com/facebook/presto/cache/CacheConfig.java

```diff
@@ -34,6 +35,10 @@
     private int maxCachedEntries = 1_000;
     private Duration cacheTtl = new Duration(2, DAYS);
 
+    //TODO: discuss - determining good default chunk size based on underlying hardware etc;
+    // is this the best way to make it configurable?
+    private int inMemoryChunkSize = (int) new DataSize(10, MEGABYTE).toBytes();
```

Comment:
Let's avoid adding a new config. 8MB can be hard coded in the manager.

Reply:
curious how you came up with 8MB?